### PR TITLE
updated docs to include the new voice isolation in teams meeting policy

### PR DESCRIPTION
**Pull request by aronee2008**

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -1114,6 +1114,23 @@ Default value: EnabledForAnyone
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
+### -VoiceIsolation
+Determines whether you provide support for your users to enable voice isolation in teams meeting calls.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:1118]:
```suggestion
Determines whether you provide support for your users to enable voice isolation in Teams meeting calls.
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 2]
```diff
@@ -1188,6 +1188,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -VoiceIsolation
+Determines whether you provide support for your users to enable voice isolation in teams meeting calls.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:1192]:
```suggestion
Determines whether you provide support for your users to enable voice isolation in Teams meeting calls.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @aronee2008, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — CHANGES_REQUESTED** [review]
parameter should also be added to parameter set section

**Comment by yogkumgit** [comment]
@aronee2008 Please implement the suggested changes. Thanks.

**Comment by aronee2008** [comment]
@yogkumgit Can you tell me where to add the changes for the parameters set section?

**Comment by get-itips** [comment]
> @yogkumgit Can you tell me where to add the changes for the parameters set section?

you can find it under SYNTAX in the article

**Review by get-itips — APPROVED** [review]

**Comment by aronee2008** [comment]
@JohanFreelancer9  can you please merge the PR if the changes look good?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 can you please merge the PR if the changes look good?

@yogkumgit

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**